test/pylib/manager_client.py

```diff
@@ -38,18 +33,11 @@ def __init__(self, sock_path: str, port: int, use_ssl: bool,
         self.con_gen = con_gen
         self.ccluster: Optional[CassandraCluster] = None
         self.cql: Optional[CassandraSession] = None
-        # API
-        self.sock_path = sock_path
-        self.sock_name = os.path.basename(sock_path)
-
-    async def start(self):
-        """Setup connection to Manager server"""
-        self.conn = aiohttp.UnixConnector(path=self.sock_path)
-        self.session = aiohttp.ClientSession(connector=self.conn)
+        self.cli = UnixRESTClient(sock_path)
```

Comment:
Again abbreviating "client" as "cli". I already pointed this out in an earlier review. Is "client" really too long?